### PR TITLE
Improvements

### PR DESCRIPTION
**Pull request by tpoisot**
A number of small improvements: the original package highlighted variables but not function, which was not really useful. Also added some recent operators, like pipes.

### CODE REVIEW

File: grammars/r.cson [hunk 1]
```diff
@@ -137,10 +148,10 @@
...
   }
-  {
-    'match': '\\b([[:alnum:]._]+)\\b'
-    'name': 'variable.other.r'
-  }
+  # {
+  #   'match': '\\b([[:alnum:]._]+)\\b'
+  #   'name': 'variable.other.r'
+  # }
   {
```
Comment (lee-dohm) [inline grammars/r.cson]: Rather than just commenting this out, it should be removed.

**Comment by lee-dohm** [comment]
Address the one comment and I'll merge it. Thanks for contributing!

**Comment by tpoisot** [comment]
Done! Thanks for porting the package (and many, many others) in the first place :+1:

**Comment by lee-dohm** [comment]
Published in v0.3.0

**Pull request merged**